.github/chainguard/enterprise-presubmit-build.sts.yaml

```diff
@@ -2,10 +2,11 @@ issuer: https://token.actions.githubusercontent.com
 # Runs presubmit and in the merge queue.
 subject_pattern: repo:chainguard-dev/enterprise-packages:.*
 claim_pattern:
-  job_workflow_ref: chainguard-dev/enterprise-packages/.github/workflows/ci-build.yaml@.*
+  job_workflow_ref: wolfi-dev/actions/.github/workflows/.ci-build.yml@.*
```

Comment:
I think we should actually switch this back and use the `workflow_ref` claim instead, which I didn't realize they'd added.  This better captures the intent here.

Reply:
done